ecosystem/sep-0006.md

```diff
@@ -67,6 +67,50 @@ Valid CORS headers are necessary to allow web clients from other sites to use th
 Access-Control-Allow-Origin: *
 ```
 
+## Recommendations
+
+SEP-6 lays out many options for how deposit and withdrawal can work. These are recommendations for getting a wallet or anchor implementation working with minimal effort while providing a great user experience.
+
+### Basic Wallet Implementation
+
+* Identify assets you want to support manually in advance
+* For each asset, use information from its `stellar.toml` and its `/info` endpoint to display useful information to the user like transaction fee structure, types of deposits and withdrawal supported, and details about the anchor like street address.
+* Use `/info` to determine:
+    * anchor endpoints that require authentication, and perform [authentication](#authentication) via SEP-10 before hitting those endpoints
+    * Fee structure: if `fee_fixed` and `fee_percent` are provided, show this to the user early in the process so they're fully informed.
+    * If the `/fee` endpoint is enabled, use it for computing fees when you need to show them to the user.
+* While `/info` allows an anchor to communicate non-standard fields that are needed for `/deposit` or `/withdraw`, it's easier for a basic wallet implemention to hard-code extra fields that are needed on a per-anchor basis, and ensure those fields are passed in properly.
+* As a conservative measure, pass in any optional fields (including amount, which you should ask the user for) for `/deposit` or `/withdraw` that the wallet has on-hand, such as `email_address` and `account`.
+* For `/deposit`
```

Comment:
Small visual suggestion: I think this would be easier to read and process if the three sections -- "use info to determine," "For deposit," and "For withdrawal" -- were more clearly delineated.  Maybe bold those?  Or make them some kind of header?